src/schema/schema_create.c

```diff
@@ -248,12 +257,14 @@ __create_file(WT_SESSION_IMPL *session, const char *uri, bool exclusive, const c
                  * To prevent mistakes with API usage, we should return an error here rather than
                  * inferring a repair.
                  */
+                WT_STAT_CONN_INCR(session, session_table_create_with_import_fail);
```

Comment:
Maybe we should increment `session_table_create_with_import_fail/session_table_create_with_import_success` in the `err:` block at the end of the method depending on the value of `ret`? Otherwise, the stats do not reflect the actual number of fails, but only fails caused by certain reasons.

Reply:
Generally I agree with this too. Having one spot where we increment one or the other at the bottom is desirable.

Reply:
I've tried to simplify this where possible, let me know if it can be further simplified.